integration_tests/lock_test.go

```diff
@@ -445,9 +445,13 @@ func (s *testLockSuite) ttlEquals(x, y uint64) {
 	// NOTE: On ppc64le, all integers are by default unsigned integers,
 	// hence we have to separately cast the value returned by "math.Abs()" function for ppc64le.
 	if runtime.GOARCH == "ppc64le" {
+		// TODO: fix the value compare
 		s.LessOrEqual(int(-math.Abs(float64(x-y))), 2)
 	} else {
```

Comment:
It seems we needn't keep this code. The problem in the note should not happen after doing a swap first.

Reply:
Removed.